tfsdk/serve_test.go

```diff
@@ -1783,7 +1821,199 @@ func TestServerPlanResourceChange(t *testing.T) {
 			resourceType:         testServeResourceTypeTwoType,
 			expectedPlannedState: tftypes.NewValue(testServeResourceTypeTwoType, nil),
 		},
-		"three_nested_computed_unknown": {
```

Comment:
NB: This was renamed to `three_nested_computed_nested_configuration_change` as seen below the next Git diff, just to capture the additional nuance between the two new test cases and the original.